mp/src/game/shared/momentum/weapon/weapon_mom_shotgun.cpp

```diff
@@ -83,13 +83,13 @@ void CMomentumShotgun::PrimaryAttack()
 #endif
         m_flPumpTime = gpGlobals->curtime + 0.5;
```

Comment:
It's good you pulled the 0.25 out into a getter, you may want to do the same for pump time.

Reply:
Consider it done